eth_tester/utils/backend_testing.py

```diff
@@ -190,6 +198,15 @@ def test_get_balance_of_listed_accounts(self, eth_tester):
             assert balance >= UINT256_MIN
             assert balance <= UINT256_MAX
 
+    def test_get_code_account_with_code(self, eth_tester):
+        emitter_address = _deploy_emitter(eth_tester)
+        code = eth_tester.get_code(emitter_address)
+        assert code != '0x'
```

Comment:
Would you mind extracting the value of `code` here and adding it to `eth_tester.utils.emitter_contract` so that we can assert the exact bytecode value.